libs/events/src/types/trade.ts

```diff
@@ -10,7 +10,16 @@ export type OnTradeParamsPayload = { orderType: UiOrderType } & Partial<{
   outputCurrencyBalance: bigint
   inputCurrencyFiatAmount: string
   outputCurrencyFiatAmount: string
-  slippageAdjustedSellAmount: bigint
+  /**
+   * Sell amount including slippage and fees.
+   * For sell orders, this value should be the same with inputCurrencyAmount
+   */
+  maximumSendSellAmount: bigint
```

Comment:
balanceSellToken - should it be `sellTokenBalance`?
Because above we have `sellToken` and `sellAmount`, just to be consistent

Reply:
Anyway, renamed other field and added units/atoms

Reply:
👍 

@alfetopito is good with names, he might have some preference too

Reply:
I like almost all suggestions, and was going to suggest the `sellTokenBalance` like Sasha did.

I do not like this one:

maximumSendSellAmount --> sellAmountLimitOrder or sellAmountAfterFees
minimumReceiveBuyAmount --> buyAmountLimitOrder or buyAmountAfterFees

I don't like the `AfterFees` because this still confuses me. We have AfterFees and BeforeFees in the backend and I never know what they mean at a first glance.

Saying `LimitOrder` is also confusing to me, given that there are other sell/buyamounts. When looking at it without knowing how it works, one might think: why is it duplicated?

I vote for keeping `maximum[Send]SellAmount`, `minimum[Receive]BuyAmount`.
Might even drop Send/Receive as the meaning is clear even without that IMO.

Reply:
Then all good, thank you!